flocker/control/test/test_protocol.py

```diff
@@ -317,12 +318,8 @@ def test_configuration_change(self):
 class FakeAgent(object):
     """
     Fake agent for testing.
-
-    Not a full verified fake since
-    https://clusterhq.atlassian.net/browse/FLOC-1255 may change this a
-    little.
     """
-    def connected(self):
+    def connected(self, client):
```

Comment:
:exclamation: This should record the client and something should test that it gets passed the correct object.